docs/en/docs/data-operate/import/import-way/spark-load-manual.md

```diff
@@ -647,6 +716,7 @@ The most suitable scenario to use spark load is that the raw data is in the file
 
 ## FAQ
 
+* Spark load does not yet support the import of Doris table fields that are of type String. If your table fields are of type String, please change them to type varchar, otherwise the import will fail, prompting `type:ETL_QUALITY_UNSATISFIED; msg:quality not good enough to cancel`
```

Comment:
```suggestion
* Spark load does not yet support the import of Doris table fields that are of type String. If your table fields are of type String, please change them to type varchar, otherwise the import will fail, prompting `type:ETL_QUALITY_UNSATISFIED; msg:quality not good enough to cancel`
```
I think this passage change as below is more better:
Spark load does not support the import of Doris table fields which is String type. If your table field has String type, please change to varchar type. Otherwise, the import will fail and prompt 'type:ETL_QUALITY_UNSATISFIED; msg:quality not good enough to cancel`